src/de/dkfz/b080/co/common/WorkflowUsingMergedBams.groovy

```diff
@@ -4,191 +4,171 @@
  * Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/COWorkflowsBasePlugin/LICENSE.txt).
  */
 
-package de.dkfz.b080.co.common
+package de.dkfz.b080.co.common;
```

Comment:
Why did you add all the `;` to a Groovy file?

Reply:
Ah that was pretty weird. I don't know why but things got mixed up. This should be the latest version of the code. And then also I had a different version of the code, which was only half done... I'll clean it up and add more comments. And yes, to your question below, it is somehow a reimport.